libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -120,7 +120,9 @@ export class ExecuteBridgeRequest {
         } else {
           // Handle unknown bridge request errors
           Logger.error(
-            `Unknown bridge request error calling \`${url}\`: \`${body}\``,
+            `Unknown bridge request error calling \`${url}\`: \`${JSON.stringify(
+              body
+            )}\``,
```

Comment:
The `body` object was not serializing to logs correctly (previously `[Object object]`), this fixes that.